### DISCUSSION TITLE
Issue with Sandcastle Help File Builder (SHFBInstaller_2024.9.12.0)

### DISCUSSION THREAD

**Discussion by Dipuna28** [EWSoftware/SHFB#1088] [Q&A]
Hello team,
We are getting issue in WPF application where we have created documentation project and using this tool and while compiling we are getting following error

**Error BUILD FAILED: An unexpected error occurred while attempting to create the build component 'Switch Component'	Documentation	SHFB	0**	

Anyone faced this issue? It would be helpful if they share the resolution for this issue.

**Comment by EWSoftware** [comment]
The log should provide more information than what you posted above which would help in diagnosing the issue.  Problems creating the components are typically caused by different versions of the SHFB tools being installed and it finding the wrong version when doing a build (installed in the usual location but then finding a local copy in the project through a NuGet package for example with each being different versions or perhaps having SHFBROOT defined in the project file and it's pointing to the wrong location).

**Reply by Dipuna28** [reply]
Hi, Thanks for the quick reply.
Can you guide us to setup the Sandle castle document in WCF application. Currently we are using visual studio 2019 

-  Microsoft Visual Studio Professional 2019 Version 16.11.42
- .NET Framework Version  4.8.09037
- SQL SERVER 2019 Version
- Sandle Castle help file builder 24.9.12.0
-

**Reply by EWSoftware** [reply]
Sorry, I don't know what you mean by "setup sandcastle document in WCF application".  What is it you are trying to do?
